shell/platform/fuchsia/flutter/compositor_context.h

```diff
@@ -35,6 +35,13 @@ class CompositorContext final : public flutter::CompositorContext {
                                float height_change_factor);
 
   void OnWireframeEnabled(bool enabled);
+  void OnCreateView(int64_t view_id, bool hit_testable, bool focusable);
+  void OnDestroyView(int64_t view_id);
+
+  flutter::ExternalViewEmbedder* GetViewEmbedder() {
+    return reinterpret_cast<flutter::ExternalViewEmbedder*>(
```

Comment:
You shouldn't need this cast at all, and it is in fact dangerous.  If we were to change the base type of SceneUpdateContext then this cast would fail silentlty and cause runtime errors

Reply:
Done